### PR TITLE
Adding help on commands

### PR DESCRIPTION
**Pull request by vincent-vandemeulebrouck-ullink**

### CODE REVIEW

File: src/main/java/commands/HelpCommandProcessor.java [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+import java.util.regex.Pattern;
+import java.util.stream.Stream;
+import com.ullink.slack.simpleslackapi.SlackChatConfiguration;
+import com.ullink.slack.simpleslackapi.SlackSession;
+import com.ullink.slack.simpleslackapi.events.SlackMessagePosted;
+
+public class HelpCommandProcessor implements SlackBotCommandProcessor
+{
+    private static final String COMMAND = "!help";
+    private static Pattern LIST_REVIEW_PATTERN = Pattern.compile(COMMAND + "(" + SPACES + "(" + ANYTHING_ELSE + "))?");
```
Comment (bcorne) [inline src/main/java/commands/HelpCommandProcessor.java]: too bad for the constant name :)

Comment (vincent-vandemeulebrouck-ullink) [reply src/main/java/commands/HelpCommandProcessor.java]: you mean rename COMMAND to HELP ?

File: src/main/java/commands/HelpCommandProcessor.java [hunk 2]
```diff
@@ -0,0 +1,103 @@
+/*************************************************************************
```
Comment (vincent-vandemeulebrouck-ullink) [inline src/main/java/commands/HelpCommandProcessor.java]: oups

**Comment by bcorne** [comment]
Many thanks for your contribution Vincent

**Pull request merged**